sample_2.py

```diff
@@ -0,0 +1,17 @@
+class Solution:
+   def fib(self, N: int) -> int:
+       if N <= 1:
+           return N
+       return self.memoize(N)
+
+   def memoize(self, N: int) -> {}:
```

Comment:
give some meanining full name to the function like getFibonacciNumber

Reply:
Variable name has been changed to `getFibonacciNumber`